### ISSUE TITLE
Cannot find module: `eslint-plugin-prettier`

### ISSUE THREAD

**Issue by ahtee** [vuejs/vue-cli#4393]
### Version
3.9.3

### Environment info
```
System:
    OS: macOS High Sierra 10.13.6
    CPU: (12) x64 Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
  Binaries:
    Node: 12.6.0 - /usr/local/bin/node
    Yarn: 1.17.3 - /usr/local/bin/yarn
    npm: 6.10.2 - /usr/local/bin/npm
  Browsers:
    Chrome: 75.0.3770.142
    Firefox: 68.0
    Safari: 12.1.2
  npmPackages:
    @vue/babel-helper-vue-jsx-merge-props:  1.0.0 
    @vue/babel-plugin-transform-vue-jsx:  1.0.0 
    @vue/babel-preset-app:  3.9.2 
    @vue/babel-preset-jsx:  1.1.0 
    @vue/babel-sugar-functional-vue:  1.0.0 
    @vue/babel-sugar-inject-h:  1.0.0 
    @vue/babel-sugar-v-model:  1.0.0 
    @vue/babel-sugar-v-on:  1.1.0 
    @vue/cli-overlay:  3.9.0 
    @vue/cli-plugin-babel: ^3.9.0 => 3.9.2 
    @vue/cli-plugin-eslint: ^3.9.0 => 3.9.2 
    @vue/cli-plugin-pwa: ^3.9.0 => 3.9.0 
    @vue/cli-service: ^3.9.0 => 3.9.3 
    @vue/cli-shared-utils:  3.9.0 
    @vue/component-compiler-utils:  2.6.0 (3.0.0)
    @vue/eslint-config-prettier: ^4.0.1 => 4.0.1 
    @vue/preload-webpack-plugin:  1.1.1 
    @vue/web-component-wrapper:  1.2.0 
    eslint-plugin-vue: ^5.0.0 => 5.2.3 (4.7.1)
    vue: ^2.6.10 => 2.6.10 
    vue-eslint-parser:  5.0.0 (2.0.3)
    vue-hot-reload-api:  2.3.3 
    vue-loader:  15.7.1 
    vue-router: ^3.0.3 => 3.0.7 
    vue-style-loader:  4.1.2 
    vue-template-compiler: ^2.6.10 => 2.6.10 
    vue-template-es2015-compiler:  1.9.1 
    vuex: ^3.0.1 => 3.1.1 
  npmGlobalPackages:
    @vue/cli: 3.9.3
```

### Steps to reproduce
Start a new vue project in a new directory: `vue create newApp`

select options: make sure to select EsLint and Prettier on the linting section

finish setup

Dependencies will install, but eslint-plugin-prettier will not be located. I dont think it is covered in @vue/eslint-config-prettier or something

See error

### What is expected?
I want to be able to use ESLint   Prettier linting config.

### What is actually happening?
I want to use the eslint   prettier setup for my project but the project errors out and I have to add eslint-plugin-prettier manually.

**Comment by Datlyfe** [comment]
same here . vue-cli v 3.9.3
in my case i had to re-install it explicitly by running `yarn add eslint-plugin-prettier --save-dev`

**Comment by sodatea** [comment]
Fixed in v3.10.
For already-created projects, see https://github.com/vuejs/vue-cli/issues/4369#issuecomment-515476598

**Comment by ahtee** [comment]
Thanks @sodatea will try again
